dsa-23au/java-dsa/jpract-app/src/test/java/com/practice/app/mergeTest.java

```diff
@@ -0,0 +1,35 @@
+package test.java.com.practice.app;
+
+
+import java.util.Random;
+
+import main.java.com.practice.app.mergeSort;
+
+public class mergeTest 
+{
+    public static void main(String[] args)
+{
+    int[] num = new int[10];
+    Random random = new Random();
+
+    int i;
+    int j;
+
+
+    for(i = 0; i < num.length; i++)
+    {
+        num[i] = random.nextInt(100000);
+        System.out.println(num[i]);
+
+    }
+
+    mergeSort MergeSort = new mergeSort();
```

Comment:
class instantiation may not be needed bc we're using the methods for the effect on the argument.  we're not updating the mergesort object state so instantiation is not needed.